core/src/test/java/com/microsoft/applicationinsights/internal/statsbeat/StatsbeatHelperTest.java

```diff
@@ -0,0 +1,42 @@
+package com.microsoft.applicationinsights.internal.statsbeat;
+
+import org.junit.Test;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static com.microsoft.applicationinsights.internal.statsbeat.Constants.JAVA_VENDOR_ZULU;
+import static org.junit.Assert.assertEquals;
+
+public class StatsbeatHelperTest {
+
+    private static final Set<String> instrumentations = new HashSet<String>() {{
+        add("io.opentelemetry.javaagent.jdbc");
+        add("io.opentelemetry.javaagent.tomcat-7.0");
+        add("io.opentelemetry.javaagent.http-url-connection");
+    }};
```

Comment:
can you change this also to static initializer? (or guava ImmutableSet if we have that on classpath here)